jest/unit/components/store/packageMeta.ts

```diff
@@ -192,7 +192,7 @@ export const packageMeta = {
     jest: { snapshotSerializers: ['jest-serializer-enzyme'] },
     engines: { node: '>=4.6.1', npm: '>=2.15.9' },
     preferGlobal: true,
-    publishConfig: { registry: 'http://localhost:4873/' },
+    publishConfig: { registry: 'https://registry.verdaccio.org' },
```

Comment:
🤔  ....

Reply:
Why a PR open with a bot ends up updating this fields? ... this should not happen.

Reply:
Hmm, I believe I pushed this mistake, but I fixed it right away